lib/global_session/rack.rb

```diff
@@ -345,6 +347,15 @@ def cookie_domain(env)
 
         domain
       end
+
+      # URL-decode a string. Some user agents like to "helpfully" URL-encode
+      # cookie and header values, and our Base64Cookie encoding scheme uses the
+      # '=' character (and expects it to cleanly round-trip). By unmunging all
+      # values before we pass them into the session serialization logic, we
+      # prevent subtle signature validation and other issues from happening.
+      def unmunge(value)
+        CGI.unescape(value)
+      end
```

Comment:
What do you think about just using `CGI.unescape(value)` directly and removing the `unmunge` method? It doesn't seem to shorten the line by much and what unmunge does is unlikely to change.